### DISCUSSION TITLE
You are not allowed to include "right" keys in your CSS.

### DISCUSSION THREAD

**Discussion by logankilpatrick** [pretalx/pretalx#1298] [Q&A] [answered]
Hey, I am getting this when I try to upload custom CSS to pretalx. Any idea what it means?

**Comment by logankilpatrick** [comment]
I searched around and there is no clear definition do "right keys" in the context of CSS.

**Reply by benabraham** [reply]
I was also looking for the list and it's to be found in

[/src/pretalx/common/css.py](https://github.com/pretalx/pretalx/blob/fda47eb4621b467820f3056a6b06f0113e64f4fe/src/pretalx/common/css.py#L54)

**Comment by rixx** [comment, marked as answer]
I assume that in your CSS, you're trying to set `right: <number> <unit>`, which pretalx does not allow.
